constants.py

```diff
@@ -1 +1,2 @@
-USERS = ["유진", "애리", "지은"]
\ No newline at end of file
+USERS = ["유진", "애리", "지은"]
+INACTIVE = ["애리"]
```

Comment:
how about use set instead of list on both `USERS` and `INACTIVE_USERS`?

Reply:
Done